platform/comm/src/main/java/com/fmicodes/comm/services/AnalyzerService.java

```diff
@@ -99,19 +99,20 @@ public ArrayList<OpenAIDestinationResponse> analyzeMessage(String message) {
     }
 
     public String getAirportIATACodeByLocation(Location location) {
-        AsyncHttpClient client = new DefaultAsyncHttpClient();
-        String airportIATACode;
-        try {
-            Response response = client.prepare("POST", "http://" + analyzerHost + "/api/v1/airports")
-                    .setHeader("Content-Type", "application/json")
-                    .setBody("{\"city\": \"" + location.getCity() + "\", \"country\": \"" + location.getCountry() + "\"}")
-                    .execute()
-                    .get();
-
-            airportIATACode = response.getResponseBody();
-        } catch (InterruptedException | ExecutionException e) {
-            throw new RuntimeException("ERROR - Communicating with Flask API: " + e.getMessage());
-        }
+        String airportIATACode = "SOF";
+
+//        AsyncHttpClient client = new DefaultAsyncHttpClient();
```

Comment:
Commented code?

Reply:
Yeah, @AntonioK180 However, we need to keep our code grounded in reality. Let's make sure we're fetching live data instead of relying on fantasy airports. @AntonioK180 I hope you learn your lesson!


I fixed that - now the commented code is uncommented and fixed, so the IATA code is fetched by the airports api. 

Resolved this and the other comment for the AnalyzerService because it is related by that.